Gems/PhysX/PhysX5/Source/Platform/Common/GCC/physx_editor_static_gcc.cmake

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(LY_COMPILE_OPTIONS
+    PUBLIC
+        -fexceptions # poly2tri 3rd Party library uses exception handling
+)
```

Comment:
Exceptions also be turned on for the clang compiler as well.
Strangely enough it is only enabled for GCC.

Reply:
Added to clang flags as well